docs/spark/how-to-guides/batch-processing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to do batch processing with .NET for Apache Spark
+description: Learn how to do batch processing using .NET for Apache Spark.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# How to do batch processing with .NET for Apache Spark
+
+This how-to shows you how to do batch processing using .NET for Apache Spark. If this is your first time using .NET for Apache Spark, check out the [Get started with .NET for Apache Spark](../tutorials/get-started.md) tutorial to learn how to prepare your environment and run your first .NET for Apache Spark application.
```

Comment:
How do we discern what's considered a how-to vs a tutorial? For ML.NET, they have guides like this one (i.e. writing an app for sentiment analysis or price prediction) in the "Tutorials" section. Is it just personal preference, or is there a standard docs guideline for how-to's vs tutorials?

Reply:
Tutorials are a step-by-step end-to-end solution. This article is not as specific and is more of a general how-to that's based on one of the four samples that are offered. We could turn this into a tutorial by adding more to the step-by-step process.